$OpenBSD: patch-libedataserver_e-debug-log_c,v 1.3 2012/09/28 06:35:36 ajacoutot Exp $

e-debug-log.c:172: warning: passing argument 1 of 'localtime' from incompatible pointer type

--- libedataserver/e-debug-log.c.orig	Sun Sep 16 17:42:01 2012
+++ libedataserver/e-debug-log.c	Thu Sep 27 20:30:27 2012
@@ -160,6 +160,7 @@ e_debug_logv (gboolean is_milestone,
 	gchar *debug_str;
 	struct timeval tv;
 	struct tm tm;
+	time_t tmp;
 
 	lock ();
 
@@ -169,7 +170,8 @@ e_debug_logv (gboolean is_milestone,
 	str = g_strdup_vprintf (format, args);
 	gettimeofday (&tv, NULL);
 
-	tm = *localtime (&tv.tv_sec);
+	tmp = tv.tv_sec;
+	tm = *localtime (&tmp);
 
 	debug_str = g_strdup_printf (
 		"%p;%04d/%02d/%02d;%02d:%02d:%02d.%04d;(%s);%s",
